templates/periodic.erb

```diff
@@ -46,3 +46,17 @@ APT::Periodic::Verbose "<%= @verbose %>";
 #      1:  progress report       (actually any string)
 #      2:  + command outputs     (remove -qq, remove 2>/dev/null, add -d)
 #      3:  + trace on
+<%- unless @random_sleep.nil? -%>
```

Comment:
Wait, I keep getting confused by this. Is it Puppet 4 safe? What does `undef` end up being in the Ruby world?